agecalculator.py

```diff
@@ -0,0 +1,69 @@
+from _datetime import datetime
+import tkinter as tk
+from tkinter import ttk
+from _datetime import *
+
+win = tk.Tk()
```

Comment:
If Object Oriented Programming practices would have been implemented it would have been much better.
Use function and if necessary classes to do the work.
It is a very unprofessional way of writing the program as a plain script.